### PR TITLE
feat: homepage ui 

### PR DESCRIPTION
**Pull request by zalborzi**
https://github.com/user-attachments/assets/999cc6a0-27df-4dc8-bf2a-3d1c24ce9acf

https://github.com/user-attachments/assets/ec238743-5c00-44ee-b219-3d1cdac14e34

## Summary by Sourcery

Update the homepage UI by introducing new fonts, enhancing component styles, and improving layout responsiveness. Replace Roboto with Quicksand and Nunito fonts, adjust text styles across components, and refine the Header and Footer for better user experience.

New Features:
- Introduce new fonts Quicksand and Nunito to the project, replacing the previously used Roboto font.

Enhancements:
- Update the Header component to use a lighter font style and improve hover effects for navigation links.
- Modify the Footer component layout to improve responsiveness and adjust text colors for better contrast.
- Refactor the HomePage component to use a consistent font style and improve text hierarchy.
- Enhance the SearchBar component by updating the input field's styling for a more modern look.
- Add organization name configuration to the contentConfig for dynamic display in the Header.

### CODE REVIEW

File: src/components/ThemesSection.tsx [hunk 1]
```diff
@@ -137,8 +137,12 @@ const ThemesSection = ({
                 width="48"
                 height="48"
               />
-              <h3 className="font-semibold text-lg sm:mt-1">{theme.name}</h3>
-              <div className="leading-4 max-w-[8.5rem]">{theme.desc}</div>
+              <h3 className="font-medium font-semibold text-lg sm:mt-1">
```
Comment (inderps) [inline src/components/ThemesSection.tsx:140]: it has to be either medium or semibold

File: src/components/Header/index.tsx [hunk 2]
```diff
@@ -103,54 +103,64 @@ function Header() {
...
-                className={`rounded-lg border-[1.5px] border-surface px-3 py-1 transition-colors duration-300 hover:border-hover-color lg:px-7 ${
-                  activeTab === "/organizations" ? "bg-primary text-white" : ""
+                className={`rounded-lg px-3 py-1 lg:px-7 transition-opacity duration-300 ${
+                  activeTab === "/organizations"
+                    ? "bg-primary text-white"
+                    : "hover:opacity-75"
                 }`}
               >
-                Organizations
+                {contentConfig.organizationName}
```
Comment (brunopacheco1) [inline src/components/Header/index.tsx:145]: As agreed, please put this back to `Organizations` and we create another story to rename everything to Publisher.

File: src/components/ThemesSection.tsx (hunk 1 above)
Comment (zalborzi) [reply src/components/ThemesSection.tsx:140]: I removed all the bold ones. It looks better now.

**Comment by zalborzi** [comment]
Uploading Screen Recording 2024-08-15 at 11.16.14.mov…

**Comment by zalborzi** [comment]
Uploading Screen Recording 2024-08-15 at 11.19.22.mov…

**Review by brunopacheco1 — APPROVED** [review]

**Pull request merged**